### ISSUE TITLE
Review Scraping does not work!!!

### ISSUE THREAD

**Issue by kani04** [facundoolano/google-play-scraper#351]
* Operating System: Windows
* Node version: v10.15.1
* google-play-scraper version: 7.0.1

### Description: 
Describe your issue here
Review Scraping returns Error

### Example code:
```js
// Put code to reproduce the issue here

var gplay = require('google-play-scraper');

gplay.reviews({
    appId: 'dji.go.v4',
    sort: gplay.sort.RATING
}).then(console.log, console.log);

```

### Error message:

```
"C:\Program Files\nodejs\node.exe" C:\Users\kani\Desktop\GooglePlayStore_Version2\sample.js
{ Error: App not found (404)
    at C:\Program Files\nodejs\node_modules\npm\node_modules\google-play-scraper\lib\utils\request.js:42:19
    at process._tickCallback (internal/process/next_tick.js:68:7) status: 404 }

Process finished with exit code 0

```

**Comment by Joepruner** [comment]
Same exact problem here.

**Comment by DaDong-G** [comment]
Same exact problem here.

**Comment by pseudozach** [comment]
looks like .reviews is requesting "https://play.google.com/store/getreviews" and that URL doesn't exist. Now they request "https://play.google.com/_/PlayStoreUi/data/batchexecute" with obfuscated json as per https://stackoverflow.com/questions/57852366/cannot-scrape-google-play-reviews-by-using-url-https-play-google-com-store-ge#comment102237874_57852366 
We need a hero :)

**Comment by icarcal** [comment]
Hey guys.
This is already solved.

You can check discussion about this at #333 and the fix at #341

This was a breaking change, so be sure to update this module to the version 7+

**Comment by kani04** [comment]
Hi. Works After updating google-play-scraper. Thank you so much @icarcal
